README.md

```diff
@@ -54,9 +79,36 @@ react-native link @react-native-community/progress-view
 
 ## Usage
 
+Import ProgressView from `@react-native-community/progress-view`
+
 ```javascript
 import {ProgressView} from "@react-native-community/progress-view";
 ```
+Add ProgressView like this
+
+```javascript
+<ProgressView
+          progressTintColor="orange"
+          trackTintColor="blue"
+          progress={0.7}
+/>
+```
+
+### Running Example App
+
+#### Windows
+1. Clone branch 
+2. cd into progress_view and run `yarn install`
+3. run `yarn add react-native@0.62 --dev`
```

Comment:
RN should already be preset after you run 'yarn install'
Didn't you mean to write RNW here?